src/smt/smt_engine.cpp

```diff
@@ -3255,14 +2886,18 @@ bool SmtEnginePrivate::simplifyAssertions()
 
     dumpAssertions("pre-nonclausal", d_assertions);
```

Comment:
Sorry, didn't see this the first time around. This can also be removed. I'll already approve the PR and mark it "do not merge".

Reply:
Same with post-nonclausal, will remove.